packages/node_modules/@webex/plugin-meetings/src/meetings/index.js

```diff
@@ -625,7 +628,7 @@ export default class Meetings extends WebexPlugin {
       // type. This must be performed prior to determining if the meeting is
       // found in the collection, as we mutate the destination for hydra person
       // id values.
-      return this.meetingInfo.fetchInfoOptions(destination, type)
+      return this.meetingInfo.fetchMeetingInfo(destination, type)
```

Comment:
hmm just to be sure: you've changed the method being called here without changing any of the code that handles the result. Does fetchMeetingInfo() return exactly the same thing that fetchInfoOptions() did? for example for v2 `options.destination` will be always undefined, right?

Reply:
Both the function in V1 and v2 will take the same options , destinations and type

There was an issue before where rather then calling the fetchMeetingInfo we were calling fetchInfoOptions which is same as fetchMeetingInfo so changed it

Reply:
but I'm talking about the code in the `then((options = {})` - line 636. `fetchMeetingInfo()` doesn't resolve with any "options", it resolves with a response body, doesn't it?